app/page.tsx

```diff
@@ -1,4 +1,9 @@
+import React from "react";
```

Comment:
We don't need this file changes because it will go into a new weekly picks page.